File: mobile_app/lib/domain/entities/geographic_line.dart
class GeographicLine {
  final String id;
  final String name;
  final String lineType;
  final String status;
  final String? phoneNumber;
  final String? carNumber;
  final String createdAt;

  const GeographicLine({
    required this.id,
    required this.name,
    required this.lineType,
    required this.status,
    this.phoneNumber,
    this.carNumber,
    required this.createdAt,
  });

  bool get isActive => status == 'active';
}

File: dashboard/src/domain/entities/GeographicLine.ts
export type LineStatus = "active" | "inactive";

export type GeographicLine = {
  id: string;
  name: string;
  lineType: string;
  contractTypeId?: string | null;
  phoneNumber?: string | null;
  carNumber?: string | null;
  vehicleId?: string | null;
  vehiclePlate?: string | null;
  phoneId?: string | null;
  phoneDisplay?: string | null;
  status: LineStatus;
  createdAt: string;
  zoneCount?: number;
};
